client/src/featureform/providers/filestore.py

```diff
@@ -0,0 +1,139 @@
+import pandas as pd
+import os
+from ..register import ScalarType
+from .online_store import OnlineStore, OnlineTable
+
+
+class LocalFileTable(OnlineTable):
+    def __init__(self, name, variant, root_path, stype):
+        self.name = name
+        self.variant = variant
+        self.root_path = root_path
+        self.dir = f"{root_path}/{name}/"
+        self.filepath = f"{root_path}/{name}/{variant}.csv"
```

Comment:
would these paths be an issue with windows?

Reply:
Dont think so